doc/geany.txt

```diff
@@ -2254,6 +2254,13 @@ Stop scrolling at last line
     When enabled Geany stops scrolling when at the last line of the document.
     Otherwise you can scroll one more page even if there are no real lines.
 
+Lines visible around the cursor
+    The number of lines to keep visible above and below the line on which the
+    cursor is when automatically scrolling the view. This allows to prevent
+    the cursor from going near the top and bottom edges in order to keep it in
+    some context. If *Stop scrolling at last line* is disabled, the cursor will
+    never reach the bottom edge when this value is greater than 0.
+
```

Comment:
"The number of lines to maintain between the cursor and the top and bottom edges of the view.  This allows some context around the cursor to always be visible. If..."